gen-apidocs/generators/html.go

```diff
@@ -79,6 +79,49 @@ func (h *HTMLWriter) WriteOverview() {
 	h.CurrentSection = &item
 }
 
+func (h *HTMLWriter) WriteAPIGroupVersions(gvs api.GroupVersions) {
+	fn := "_group_versions.html"
+	path := filepath.Join(api.IncludesDir, fn)
+	f, err := os.Create(path)
+	defer f.Close()
+	if err != nil {
+		os.Stderr.WriteString(fmt.Sprintf("%v", err))
+		os.Exit(1)
+	}
+
+	fmt.Fprintf(f, h.DefaultStaticContent("API Groups"))
+	fmt.Fprintf(f, "<P>The API Groups and their versions are summarized in the following table.</P>")
+	fmt.Fprintf(f, "<TABLE class=\"col-md-8\">\n<THEAD><TR><TH>Group</TH><TH>Version</TH></TR></THEAD>\n<TBODY>\n")
```

Comment:
Can we show what kind of format we have so that we need to print this formatting? Or probably write the expected output as comment on the function.

Reply:
See my comment above. It is a screenshot